oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -38,6 +39,12 @@
     private int superDatasetIndexShardsFactor = 5;
     @Setter
     private int indexReplicasNumber = 0;
+    /**
+     * @since 8.2.0, the replicas number is for super size dataset record replicas number
+     * @see SuperDataset
+     */
+    @Setter
+    private int superDatasetIndexReplicasNumber = 0;
```

Comment:
As you moved all superset config in one place in the doc and config file, please do the same thing for the config class

Reply:
done